src/components/bcros/businessDetails/Links.vue

```diff
@@ -182,7 +188,16 @@ const dissolveBusiness = async (): Promise<void> => {
       @close="showDissolutionDialog(false)"
     >
       <template #content>
-        <div>
+        <div v-if="!currentBusiness.goodStanding">
```

Comment:
The Not In Good Standing dialog is displayed when a business is not in good standing and the user is not staff. https://github.com/bcgov/business-filings-ui/blob/5cc4b75655897ad604706672eea61cde9e9609aa/src/components/EntityInfo/EntityMenu.vue#L438C1-L444C4